pkg/adapter/nimbus-k8tls/manager/manager.go

```diff
@@ -31,7 +31,7 @@ import (
 var (
 	scheme        = runtime.NewScheme()
 	k8sClient     client.Client
-	NamespaceName = "nimbus-k8tls-env"
+	NamespaceName string
```

Comment:
I don't think we should make namespace configurable in which we deploy the k8tls related stuff. So we should use hardcoded namespace.

```suggestion
	NamespaceName = "nimbus-k8tls-env"
```
or 
```suggestion
	NamespaceName = "k8tls-env"
```